### ISSUE TITLE
If a theme is applied, use the theme's token in the `LoadingProvider` component

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1100]
### Which package has an issue?

@yamada-ui/react

### Package Version

1.3.0

### Description

Currently, styling values in components ignore theme tokens. If a theme is applied, you should use the theme's token.

Yamada UI's style system provides fallback functions. If the value of the token in the first argument does not exist, it returns the second argument, the fallback value.

```diff
const css: CSSUIObject = {
  bg: ["fallback(white, #fbfbfb)", "fallback(black, #141414)"],
  maxW: "24rem",
-  p: "1rem",
+  p: "fallback(4, 1rem)",
  display: "flex",
  flexDirection: "column",
  justifyContent: "center",
  alignItems: "center",
-  gap: "0.5rem",
+  gap: "fallback(2, 0.5rem)",
-  rounded: "0.375rem",
+  rounded: "fallback(md, 0.375rem)",
-  boxShadow: [
-    "0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)",
-    "0px 0px 0px 1px rgba(0, 0, 0, 0.1), 0px 5px 10px rgba(0, 0, 0, 0.2), 0px 15px 40px rgba(0, 0, 0, 0.4)",
-  ],
+  boxShadow: [
+    "fallback(lg, 0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05))",
+    "fallback(dark-lg, 0px 0px 0px 1px rgba(0, 0, 0, 0.1), 0px 5px 10px rgba(0, 0, 0, 0.2), 0px 15px 40px rgba(0, 0, 0, 0.4))",
+  ],
}
```

```diff
const getOverlayStyle = (
  type: "fill" | "transparent" = "fill",
): CSSUIObject => ({
  position: "fixed",
  top: 0,
  right: 0,
  bottom: 0,
  left: 0,
-  zIndex: 9999,
+  zIndex: "fallback(full, 9999)",
  bg:
    type === "fill"
      ? ["fallback(white, #fbfbfb)", "fallback(black, #141414)"]
      : "fallback(blackAlpha.600, rgba(0, 0, 0, 0.48))",
  w: "100vw",
  h: "100dvh",
-  p: "1rem",
+  p: "fallback(4, 1rem)",
  display: "flex",
  justifyContent: "center",
  alignItems: "center",
})
```

```diff
const css: CSSUIObject = {
  position: "fixed",
-  right: "1rem",
+  right: "fallback(4, 1rem)",
-  bottom: "1rem",
+  bottom: "fallback(4, 1rem)",
-  zIndex: 9999,
+  zIndex: "fallback(full, 9999)",
  bg: ["fallback(white, #fbfbfb)", "fallback(black, #141414)"],
  maxW: "20rem",
-  p: "0.5rem",
+  p: "fallback(2, 0.5rem)",
  display: "flex",
  justifyContent: "center",
  alignItems: "center",
-  gap: "0.5rem",
+  gap: "fallback(2, 0.5rem)",
-  rounded: "0.375rem",
+  rounded: "fallback(md, 0.375rem)",
-  boxShadow: [
-    "0 25px 50px -12px rgba(0, 0, 0, 0.25), 0 -25px 50px -12px rgba(0, 0, 0, 0.25)",
-    "0px 0px 0px 1px rgba(0, 0, 0, 0.1), 0px 5px 10px rgba(0, 0, 0, 0.2), 0px 15px 40px rgba(0, 0, 0, 0.4)",
-  ],
+  boxShadow: [
+    "fallback(3xl, 0 25px 50px -12px rgba(0, 0, 0, 0.25), 0 -25px 50px -12px rgba(0, 0, 0, 0.25))",
+    "fallback(dark-lg, 0px 0px 0px 1px rgba(0, 0, 0, 0.1), 0px 5px 10px rgba(0, 0, 0, 0.2), 0px 15px 40px rgba(0, 0, 0, 0.4))",
+  ],
}
```

```diff
const css: CSSUIObject = {
  maxW: "24rem",
  display: "flex",
  flexDirection: "column",
  justifyContent: "center",
  alignItems: "center",
-  gap: "0.5rem",
+  gap: "fallback(2, 0.5rem)",
}
```

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Browser

None

### Operating System

- [ ] macOS
- [ ] Windows
- [ ] Linux

### Do you know how to fix the issue?

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix?

None

**Comment by minagishl** [comment]
@hirotomoyamada 
Can I work on this issue?

**Comment by hirotomoyamada** [comment]
@minagishl 

Hi, thanks for the issue!

Yes, please.

I'll assign this issue to you!

**Comment by taroj1205** [comment]
Fixed with #1108
